File: Java/src/DS/MultiLevelListNode.java
package DS;

public class MultiLevelListNode {
    public int val;
    public MultiLevelListNode next;
    public MultiLevelListNode child;
    public MultiLevelListNode(int val) { this.val = val; }
    public MultiLevelListNode(int val, MultiLevelListNode next, MultiLevelListNode child) {
        this(val);
        this.next = next;
        this.child = child;
    }
}

File: csharp/Linked Lists/MultiLevelListNode.cs
public class MultiLevelListNode
{
    public MultiLevelListNode(
        int val, MultiLevelListNode next = null, MultiLevelListNode child = null)
    {
        Val = val;
        Next = next;
        Child = child;
    }

    public int Val { get; set; }

    public MultiLevelListNode Next { get; set; }
    
    public MultiLevelListNode Child { get; set; }
}
